openpype/hosts/houdini/api/shelves.py

```diff
@@ -0,0 +1,195 @@
+import os
+import logging
+import platform
+
+from openpype.settings import get_project_settings
+
+import hou
+
+log = logging.getLogger("openpype.hosts.houdini")
+
+
+def generate_shelves():
+    """This function generates complete shelves from shelf set to tools
+    in Houdini from openpype project settings houdini shelf definition.
+
+    Raises:
+        FileNotFoundError: Raised when the shelf set filepath does not exist
+    """
+    current_os = platform.system().lower()
+
+    # load configuration of houdini shelves
+    project_settings = get_project_settings(os.getenv("AVALON_PROJECT"))
+    shelves_set_config = project_settings["houdini"]["shelves"]
+
+    if not shelves_set_config:
+        log.info(
+            "SHELF INFO: No custom shelves found in project settings."
+        )
```

Comment:
Should this just be a debug message?
Also I steel feel like doing `log.info` and also including "INFO" in the message is kind of saying the same things twice.

I'd expect something like this:
```suggestion
        log.debug("SHELF: No custom shelves found in project settings.")
```

And if the logger at the top would be:
```python
log = logging.getLogger("openpype.hosts.houdini.shelves")
```
(Or anything that at least hints that this logger relates to the shelves)...
Then I'd say the logged message could even be:

```suggestion
        log.debug("No custom shelves found in project settings.")
```

It's the same with the other logged messages that include INFO, WARNING or ERROR in the message itself - shouldn't the logging level (`log.info`, `log.warning`, `log.error`) describe that itself?